docs/user-guide/incident-commander.md

```diff
@@ -6,61 +6,104 @@ description: What to expect as an incident commander.
 
 ## Reporting
 
-Within Dispatch Incident Commander's \(ICs\) are also participants and will receive all of the messaging that a participant does. When resolved as incident commander you are assigned that role by Dispatch and your identity is propagated via various messaging. Additionally for `High` severity events, the Incident Commander is automatically paged \(via PagerDuty\).
+Within Dispatch, Incident Commanders \(ICs\) are also participants, and will receive all of the participant messaging as well. When resolved as the Incident Commander, you are assigned that role by Dispatch, and your identity is propagated via various messaging. For `High` priority incidents, the Incident Commander is automatically paged \(via PagerDuty\).
 
-## During
+All Slack commands are listed below, or you may view _groups_ of commands relating to [People](#people), [Communications](#communications), [Tasks](#tasks), and [Incident Resources & Metadata](#incident-resources-and-metadata).
```

Comment:
Should we point people to the list of commands [here](https://hawkins.gitbook.io/dispatch/configuration/plugins/configuring-slack#commands), which are up to date,  rather than listing them again here, which doubles the amount of maintenance?

Reply:
Yeah I'm a fan of DRY so that's a great idea. Erm... and... to do that we should recognize that these serve two different purposes. One seeks to educate incident commanders on how and why to use the commands in Slack, and the other helps people when configuring the app. Seems like we could either merge these, or double down on a separate section to help people use the software after configuration. Do you anticipate a lot of further changes to our commands? That could change how we approach it.

Reply:
Yeah the User Guide is definitely a different audience from those configuring the app. Thinking that a little duplication may just be a necessary evil in this case, to help users. I hope to anticipate and head off support questions, you know?

Reply:
Sounds good.